README.md

```diff
@@ -134,6 +134,12 @@ See [here](https://github.com/Azure/azure-functions-core-tools/blob/master/READM
 
 * [Python 3.6.x](https://www.python.org/downloads/)
 
+### PowerShell
```

Comment:
Also add a link to the PowerShell section here: https://github.com/Microsoft/vscode-azurefunctions/blob/master/README.md#prerequisites

Reply:
done